projects/Rockchip/devices/RK3328/README.md

```diff
@@ -4,12 +4,9 @@ This is a SoC device for RK3328
 
 **Build**
 
-* `PROJECT=Rockchip DEVICE=RK3328 ARCH=arm UBOOT_SYSTEM=box make image`
```

Comment:
what does mean `currently supported boards` ? all other boards are now not working anymore ?

Reply:
They don't have upstream devicetrees (I guess they were still in from le 9.2 times) - the images built until now form them had no devicetee in extlinux-partition.

Reply:
what happens if you update from 9.2 -> 10 with such a device then ?

Reply:
Either somebody steps up and creates devicetrees for all devices that were supported in 9.2 - and also upstreams them or we shoudn't allow le 9.x -> 10 upgrades (as other projects also do). 
I can ad the latter one to this PR: @Kwiboo: your decision

Reply:
if we can't fix the device trees we should disallow upgrades, user will not know about it and won't read warning text so better protect them

Reply:
Done. Thanks for the heads-up.

Reply:
The DTs are missing since bc55a55cbbb91e3fbc38cd52e73436def957772d. I could check what it takes to add them again, if that is the idea.

Reply:
This compiles: https://github.com/easybe/linux/commits/le_dts
Not sure what it will take to upstream though.